templates/pageLevelProgressIndicator.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { compile } from 'core/js/reactHelpers';
+
+export default function PageLevelProgressIndicator (props) {
+  const {
+    ariaLabel
+  } = props;
+
+  return (
+    <span className='pagelevelprogress__indicator'>
+      <span className="pagelevelprogress__indicator-inner">
+
+        <span className="pagelevelprogress__indicator-bar"></span>
+
+        {ariaLabel &&
+        <span className="aria-label">
+          {compile(ariaLabel)}
```

Comment:
The ariaLabel currently doesn't include the percentageComplete, pass through props so percentageComplete is included. 

```suggestion
          {compile(ariaLabel, props)}
```